guides/spin/cli/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Install and Configure Spin CLI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Install `spin`
+
+One way to manage applications and pipelines as code is through [spin](http://github.com/spinnaker/spin).
+
+To acquire `spin`, do the following:
+
+```bash
+# See the versions released at https://github.com/spinnaker/spin/releases.
+export VERSION=<desired spin version tag> # Select a tag from the released version, e.g. v0.1.0
+curl -L https://github.com/spinnaker/spin/releases/download/$VERSION/spin -o spin
+chmod +x ./spin
+./spin --help
+```
+
+**Note**: The alpha `spin` binaries are only available for Linux platforms only at this point.
+We'll compile binaries for other platforms likely in Spinnaker 1.9.x.
+
+## Configure `spin`
+
+`spin` reads its configuration from `~/.spin/config`. At this point in time, all of its configuration
+is for authentication mechanisms.
```

Comment:
Similarly here, I'd like to delete "At this point in time,..."

But if we really want to say that we've only put in config for authentication mechanisms so far, but there's more to come, then let's condense down to...

"Currently, all configuration is for authentication only."